samples/graph-appcatalog-lifecycle/nodejs/README.md

```diff
@@ -0,0 +1,89 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
```

Comment:
This looks like additional readme file for App Catalog, could you please remove it?